ADALiOS/ADALiOS/ADAuthenticationContext+WebRequest.m

```diff
@@ -295,9 +296,9 @@ - (NSString*)queryStringFromResource:(NSString*)resource
     
     [startUrl appendFormat:@"&%@", [[ADLogger adalId] adURLFormEncode]];
     
-    if (![NSString adIsStringNilOrBlank:userId])
+    if (![NSString adIsStringNilOrBlank:userId.userId])
     {
-        [startUrl appendFormat:@"&%@=%@", OAUTH2_LOGIN_HINT, [userId adUrlFormEncode]];
+        [startUrl appendFormat:@"&%@=%@", OAUTH2_LOGIN_HINT, [userId.userId adUrlFormEncode]];
```

Comment:
If you look at the UserIdentifier class in .NET (https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/blob/master/src/ADAL.Common/UserIdentifier.cs) the only fields are 'id' and 'type.' I can't call a field in ObjC 'id' because that's a reserved word in the language so I used 'userId' instead. It doesn't map to 'userId' concept in ADUserInformation, however that userId concept isn't in the .NET client either.